nvflare/private/fed/server/job_cmds.py

```diff
@@ -79,9 +79,19 @@ def authorize_job(self, conn: Connection, args: List[str]):
 
         job_id = args[1].lower()
         conn.set_prop(self.RUN_NUMBER, job_id)
-        args.append("server")  # for now, checking permissions against server
+        engine = conn.app_ctx
+        job_def_manager = engine.job_def_manager
```

Comment:
note that if this engine does not have `job_def_manager`
this L83 will raise `AttributeError`, we don't get to L84

Reply:
AttributeError is just as meaningful so I am going to remove this check.